### PR TITLE
80 header

### LINKED ISSUE #80: Header component
Create header component

### PR DESCRIPTION
**Pull request by Mahmoud-Ahmad2**
Relates #80  
Close #80

### CODE REVIEW

**Comment by Mahmoud-Ahmad2** [comment]
Screen Shot 2022-05-20 at 6 34 56 PM

File: client/src/components/Header/index.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            </p>
+            {!user ? (
+              <Button className="btnStartPracticing" type="primary">
+                Start Practicing
+              </Button>
+            )
+              : (
+                <Button className="btnBookInterview" type="primary">
+                  Book Interview
+                </Button>
```
Comment (Raghad-Mezied) [inline client/src/components/Header/index.jsx:40]: Link these btn

File: client/src/components/Header/index.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  const { user } = useSelector((state) => state.auth);
+  return (
+    <Layout>
+      <Content
+        className="site-layout-background headerContent"
+        style={{
+          padding: 24,
+          margin: 0,
+          minHeight: 280,
+        }}
```
Comment (Raghad-Mezied) [inline client/src/components/Header/index.jsx:20]: Move the style to the css file

File: client/src/components/Header/index.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        <div className="allSec">
+          <section className="rightSec">
+            <div className="novaCommunity">
+              <p className="pNova">Nova Community</p>
+            </div>
+            <h1 className="PracticeMakesPerfect">Practice Makes Perfect</h1>
+            <p className="pDescribe">
+              More than 2 billion people in over countries use Nova to
+              practice with professional interviewer.
+            </p>
```
Comment (Raghad-Mezied) [inline client/src/components/Header/index.jsx:31]: Check Typography antd component

**Review by Raghad-Mezied — CHANGES_REQUESTED** [review]

**Review by manar-abed — APPROVED** [review]

**Review by Raghad-Mezied — APPROVED** [review]

**Review by abedshamia — APPROVED** [review]

**Pull request merged**